testfiles/AirflowNetwork3zVent.idf

```diff
@@ -743,9 +743,23 @@
     ,                        !- Shading Control Name
     ,                        !- Frame and Divider Name
     1.0,                     !- Multiplier
-    4,                       !- Number of Vertices
+    3,                       !- Number of Vertices
     1.00000,0,2.500000,  !- X,Y,Z ==> Vertex 1 {m}
     1.00000,0,1.0000000,  !- X,Y,Z ==> Vertex 2 {m}
+    5.000000,0,1.0000000;  !- X,Y,Z ==> Vertex 3 {m}
```

Comment:
@jasondegraw Thanks for finding a mistake. The correct coordinates have been changed. The window view is shown below.

![window](https://cloud.githubusercontent.com/assets/5970656/20971038/01ee5332-bc5e-11e6-8e12-3bfc007a0dc7.JPG)
 
I prefer to add notes, instead of adding one more example file, since we already have so many example files. Here are notes I would like to add under Highlights in the file:

A rectangular window was split into two triangular windows to demonstrate how the AirflowNetwrok model handles non-rectangular windows. Rectangular windows are recommended to be used in most cases.

Please let me know what you think.